### ISSUE TITLE
Source level Mappings return incorrect response headers and no facets

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/oclapi#130]
`GET /orgs/CIEL/sources/CIEL/mappings/`does not return the correct response headers and facets are not included when `includeFacets: true` is set in the request header.

NOTE: Can only test this on dev and staging right now due to 404 long-running query error on production (due to lots of mappings being loaded)

Here is a sample of the response header returned (same regardless of the setting of `includeFacets`):
```
Server: nginx/1.4.6 (Ubuntu)
Date: Mon, 13 Jul 2015 19:53:23 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Connection: keep-alive
Vary: Accept
Allow: GET, POST, HEAD, OPTIONS
```

Here is an example of what should be returned:
```
Server: nginx/1.4.6 (Ubuntu) 
Date: Fri, 10 Jul 2015 11:07:33 GMT 
Content-Type: application/json; charset=utf-8 
Transfer-Encoding: chunked 
Connection: keep-alive
Vary: Accept 
next: http://api.dev.openconceptlab.com/orgs/CIEL/sources/CIEL/mappings/?page=2
num_found: 1103
Allow: GET, POST, HEAD, OPTIONS 
offset: 0
num_returned: 10
previous: None
```

**Comment by MisterNando** [comment]
Part of this (the response headers) was resolved with #127; the other part is resolved in the associated commit (above).

**Comment by paynejd** [comment]
Tested-- works correctly!
